client/blocks/location-search/index.jsx

```diff
@@ -0,0 +1,98 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { loadScript } from 'lib/load-script';
+import config from 'config';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import SearchCard from 'components/search-card';
+
+let autocompleteService = null;
+
+class LocationSearch extends Component {
+	static propTypes = {
+		onPredictionClick: PropTypes.func,
+		predictionHref: PropTypes.string,
+	};
+
+	state = { predictions: [] };
+
+	componentDidMount() {
+		if ( ! autocompleteService ) {
+			autocompleteService = {};
```

Comment:
I don't see why we need to initialize this variable to an empty object. Could you clarify that for me?

Reply:
I've added a comment to explain